File: POSmaisgestao/app/src/main/java/com/pos_mais_gestao/hardware/balanca/protocol/Prt3Protocol.java
package com.pos_mais_gestao.hardware.balanca.protocol;

public final class Prt3Protocol {
    public static final byte ENQ = 0x05;
    public static final byte STX = 0x02;
    public static final byte ETX = 0x03;

    private Prt3Protocol() {}

    public static byte[] solicitarPeso() {
        return new byte[] {ENQ};
    }
}

File: estacao-balanca/lib/features/balanca/prt3_protocol.dart
/// Protocolo Toledo Prix Prt3 (C14=Prt3, C15=2400, 8N1).
class Prt3Protocol {
  static const int enq = 0x05;
  static const int stx = 0x02;
  static const int etx = 0x03;

  static List<int> solicitarPeso() => [enq];
}
